samples/apps/copilot-chat-app/webapi/DeploymentTemplates/main.bicep

```diff
@@ -135,149 +128,154 @@ resource appServiceWeb 'Microsoft.Web/sites@2022-03-01' = {
-          value: deployNewAzureOpenAI ? openAI.properties.endpoint : endpoint
-        }
-        {
-          name: 'Planner:AIService:Key'
-          value: deployNewAzureOpenAI ? openAI.listKeys().key1 : apiKey
-        }
-        {
-          name: 'Authorization:Type'
-          value: empty(skServerApiKey) ? 'None' : 'ApiKey'
-        }
-        {
-          name: 'Authorization:ApiKey'
-          value: skServerApiKey
-        }
-        {
-          name: 'ChatStore:Type'
-          value: deployCosmosDB ? 'cosmos' : 'volatile'
-        }
-        {
-          name: 'ChatStore:Cosmos:Database'
-          value: 'CopilotChat'
-        }
-        {
-          name: 'ChatStore:Cosmos:ChatSessionsContainer'
-          value: 'chatsessions'
-        }
-        {
-          name: 'ChatStore:Cosmos:ChatMessagesContainer'
-          value: 'chatmessages'
-        }
-        {
-          name: 'ChatStore:Cosmos:ConnectionString'
-          value: deployCosmosDB ? cosmosAccount.listConnectionStrings().connectionStrings[0].connectionString : ''
-        }
-        {
-          name: 'MemoriesStore:Type'
-          value: deployQdrant ? 'Qdrant' : 'Volatile'
-        }
-        {
-          name: 'MemoriesStore:Qdrant:Host'
-          value: deployQdrant ? 'http://${aci.properties.ipAddress.fqdn}' : ''
-        }
-        {
-          name: 'AzureSpeech:Region'
-          value: location
-        }
-        {
-          name: 'AzureSpeech:Key'
-          value: deploySpeechServices ? speechAccount.listKeys().key1 : ''
-        }
-        {
-          name: 'Kestrel:Endpoints:Https:Url'
-          value: 'https://localhost:443'
-        }
-        {
-          name: 'Logging:LogLevel:Default'
-          value: 'Warning'
-        }
-        {
-          name: 'Logging:LogLevel:SemanticKernel.Service'
-          value: 'Warning'
-        }
-        {
-          name: 'Logging:LogLevel:Microsoft.SemanticKernel'
-          value: 'Warning'
-        }
-        {
-          name: 'Logging:LogLevel:Microsoft.AspNetCore.Hosting'
-          value: 'Warning'
-        }
-        {
-          name: 'Logging:LogLevel:Microsoft.Hosting.Lifetimel'
-          value: 'Warning'
-        }
-        {
-          name: 'ApplicationInsights:ConnectionString'
-          value: appInsights.properties.ConnectionString
-        }
-        {
-          name: 'APPLICATIONINSIGHTS_CONNECTION_STRING'
-          value: appInsights.properties.ConnectionString
-        }
-        {
-          name: 'ApplicationInsightsAgent_EXTENSION_VERSION'
-          value: '~2'
-        }
-      ]
-    }
   }
   dependsOn: [
     logAnalyticsWorkspace
   ]
 }
 
+resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
+  parent: appServiceWeb
+  name: 'web'
+  properties: {
+    alwaysOn: true
+    detailedErrorLoggingEnabled: true
+    minTlsVersion: '1.2'
+    netFrameworkVersion: 'v6.0'
+    use32BitWorkerProcess: false
+    appSettings: [
+      {
```

Comment:
We'll need to update these to reflect the optimizations from here: https://github.com/microsoft/semantic-kernel/pull/1031